source/lib/src/prod_force.cc

```diff
@@ -36,14 +36,17 @@ prod_force_a_cpu(
 
   memset(force, 0.0, sizeof(FPTYPE) * nall * 3);
   // compute force of a frame
+  #pragma omp parallel
```

Comment:
Two threads with different i_idx may write on the same force `force[j_idx * 3 + xxx]`

Reply:
I realized that, so that `pragma omp for` is inside this loop and before another loop. The current version can pass the tests.

Reply:
OK I understand you compute the neighbors of the same atom using multi-threading. 

What I do not understand is why you need `omp parallel` to generate threads here, but not at L49 where you really need multi-threading

Reply:
[![image](https://user-images.githubusercontent.com/9496702/146475453-837f49b1-3484-4727-b38a-fa268f806d0e.png)](https://www.openmp.org/wp-content/uploads/openmp-webinar-vanderPas-20210318.pdf)